skrf/__init__.py

```diff
@@ -3,10 +3,9 @@
 implemented in Python.
 """
 
-__version__ = '1.0.0'
+__version__ = "1.0.0"
```

Comment:
Out of curiosity: why the change from ' to " for the strings? I always thought Python does accept both notations. Is it personal taste or a new general rule?

Reply:
That's almost certainly ruff's auto formatting. I try to avoid noisy commits that are primarily changes due to auto formatting but sometimes stuff slips by 😮‍💨

Reply:
That's almost certainly ruff's auto formatting. I try to avoid noisy commits that are primarily changes due to auto formatting but sometimes stuff slips by 😮‍💨